ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2601,6 +2601,10 @@ router_general:
           subscribe_policy: < route-map policy >
         - source_vrf: < source-vrf >
           subscribe_policy: < route-map policy >
+      routes:
+        dynamic_prefix_lists:
```

Comment:
EOS supports several other knobs on this:
`routes dynamic prefix-list test install drop preference 12`
I think we need to make the list items into dicts, with a `name` key for prefix_list name and then later we can add other knobs.

Reply:
Thanks for the insight.  My EOS devices / versions did not show those knobs.